config/projects/angrychef.rb

```diff
@@ -47,6 +47,17 @@
 compress :dmg
 
 package :msi do
+  install_path = Pathname.new(install_dir)
+
+  # Find path in which the Chef gem is installed
+  search_pattern = "#{install_path}/**/gems/chef-[0-9]*-mingw32/ext/win32-eventlog/chef-log.dll"
+  dll_path  = Pathname.glob(search_pattern).find { |path| path.file? }
```

Comment:
There wasn't an easy place to refactor it :(  I could put it in the MSI packager but that seems overly specific.  Omnibus util seems like a good place but even there, the particular strategy of using a random path glob/search pattern seemed sub-optimal.  What would really be nice is if I could interrogate the software definition itself to see the list of gems it installed or being able to run "gem which".  I do have an item to refactor this code/handle it better - for now, it follows the current workaround pattern in push-jobs.